src/ops/init/mod.rs

```diff
@@ -0,0 +1,37 @@
+//! Bootstrap a new lorri project
+
+use crate::ops::{ok, ExitError, OpResult};
+use std::fs::File;
+use std::io;
+use std::io::Write;
+use std::path::Path;
+
+fn create_if_missing(path: &Path, contents: &str) -> Result<(), io::Error> {
+    if path.exists() {
+        println!("- Skipping existing {}", path.display());
+        Ok(())
+    } else {
+        let mut f = File::create(path)?;
+        f.write_all(contents.as_bytes())?;
+        println!("- Writing {}", path.display());
+        Ok(())
+    }
+}
+
+fn to_op(e: Result<(), io::Error>) -> OpResult {
+    match e {
+        Ok(_) => ok(),
+        Err(e) => ExitError::errmsg(format!("{}", e)),
+    }
+}
+
+/// See the documentation for lorri::cli::Command::Init for
+/// more details
+pub fn main(default_shell: &str, default_envrc: &str) -> OpResult {
+    println!("Bootstrapping a new lorri project\n");
```

Comment:
I think we can leave out this message, because the user ostensibly requests the bootstrap when she types `lorri init`.

Reply:
Or, or leave it in, but use it to print the directory: `"… in /home/user/current/working/dir"`.

Reply:
Actually I think you were right - I agree that I should just remove it